src/main.js

```diff
@@ -30,6 +31,16 @@ Vue.use(VueHead, {
 	complement: "Popcorn.moe"
 });
 
+// vue-timeago
+Vue.use(VueTimeago, {
+	name: "timeago",
+	locale: "en",
+	locales: {
+		fr: require("date-fns/locale/fr"),
+		en: require("date-fns/locale/en")
```

Comment:
I don't know if there is a way to dynamically import locales (so we don't make the user load unnecessary locales)